scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -681,9 +701,17 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
 
     @tailrec
     def countIndentInternal(pos: Int, acc: Int = 0): (Int, Int) = {
-      if (pos < 0 || scannerTokens(pos).is[LF]) (acc, pos)
-      else if (isWhitespace(scannerTokens(pos))) countIndentInternal(pos - 1, acc + 1)
-      else (-1, -1)
+      val token = scannerTokens(pos)
```

Comment:
should you access token before checking `pos < 0`?

Reply:
@dos65 a bug?

Reply:
yep. thx!